src/constants/appConstants.tsx

```diff
@@ -410,3 +410,42 @@ export const SESSIONS_TABS = [
   { header: SECONDARY },
   { header: BOTH },
 ];
+
+export const QUICK_LINKS = [{
+  id: 0,
+  header: "Alerting & Monitoring Tools",
+  links: [
+    { name: "Azure Splunk", link: "http://azruvapspl11.sephoraus.com:8000/en-US/app/atg/search" },
```

Comment:
Keep all these links as environment variables and import accordingly.

Reply:
done